src/DataSkimmer.jl

```diff
@@ -92,11 +93,29 @@ struct DateTimeColumn
     end
 end
 
+struct AllMissingColumn
+    name::Symbol
+    type::Type
+    n_missing::Int64
+    completion_rate::Float64
```

Comment:
The all missing columns will always have `n_missing == n_rows` and `completion_rate` of 100%, so I think these properties can be removed.

This will make the output for allmissing columns quite boring, but they're boring columns anyway! :sweat_smile:

Reply:
All set. Needed to update the `Base.show` method for `Vector{AllMissingColumn}` as well.

Reply:
Formatter run without trouble. Let me know if anything else needed.